conda/recipes/dask-cuml/meta.yaml

```diff
@@ -32,27 +32,17 @@ build:
 
 requirements:
   build:
-    - python
-    - cudf 0.7*
-    - dask >=0.19.0
     - dask-cudf 0.7*
     - dask-cuda 0.7*
   run:
-    - python
-    - cudf 0.7*
-    - dask >=0.19.0
```

Comment:
Would also keep `python` and `dask` here.

Reply:
I thought the indirect dependency of python(or dask) through dask-cudf is good enough. Any specific reason why we still want to add them explicitly ?

Reply:
It's best practice to list explicit requirements. We are using `dask` in this library explicitly.

Also we need to list everything in `run` that we expect to be installed with this package. That includes `python`.

ref: https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/building-conda-packages.html#edit-the-skeleton-files